feature/main/src/main/java/com/ku_stacks/ku_ring/main/notice/compose/inner_screen/CategoryNoticeScreen.kt

```diff
@@ -77,6 +79,8 @@ private fun CategoryNoticeScreenContents(
                 .pullRefresh(refreshState),
             // TODO: 일단 비중요 공지만 보여주고, 나중에 정책이 결정되면 다시 수정하기
             noticeFilter = { !it.isImportant },
+            // TODO: see CategoryNoticeViewModel.insertNoticeToLocal
```

Comment:
TODO를 적을때 누가 TODO를 적었는지

```suggestion
            // TODO by MW see CategoryNoticeViewModel.insertNoticeToLocal
```

이런식으로 투두적어보는 건 어떨까요? 그러면 관련 이슈 논의할 때 더 편할 것 같아요

Reply:
좋습니다! 저는 대충 mwy 정도로 적으면 되겠네요. 이건 컨벤션으로 해도 될 것 같습니다

tmi: 사실 지금 적혀있는 TODO는 전부 제가 적었어요 ㅋㅋ